VELS_WEB/controllers/vels_plugins.py

```diff
@@ -25,36 +29,70 @@ def __list_plugins():
 
     return returnDict
 
-def __assemble_plugin(plugin_name):
+def __assemble_plugin(plugin_name, returnDict):
 
     #####
     # set entries for each plugin
     # plugin_active controls if plugin is visible at VELS_WEB
     # plugin_active has to be set for every possible plugin
     #####
-    returnDict={}
 
-    #if(plugin_name == 'vels_ob'):
-     #   returnDict.update({'vels_ob_active': True})
+    if(plugin_name == 'vels_ob'):
+        returnDict.update({'vels_ob_active': True})
+        
+        MelodiURL = returnDict.get("PluginParameter").get(plugin_name).get("server")
+        MelodiPort = returnDict.get("PluginParameter").get(plugin_name).get("port")
+
+        api_con = api_vels_ob.Configuration()
+        api_con.host = "{}:{}/api".format(MelodiURL, MelodiPort)
```

Comment:
Alternative would be f-string, since Python 3.6:
api_con.host = f"{MelodiURL}:{MelodiPort}/api"